libraries/ui/src/utils/addQueryParam.test.ts

```diff
@@ -0,0 +1,38 @@
+import { describe, test, expect } from 'vitest';
+import { addQueryParam } from './addQueryParam';
+
+describe('addQueryParam', () => {
```

Comment:
would be curious what behaviour is when:
- submitting an invalid URL (I think this throws - reasonable)
- adding a parameter that already exists on the URL (not sure what is reasonable to do here - maybe overwrite?)

Reply:
Small followup in https://github.com/bluedotimpact/bluedot/pull/741